### ISSUE TITLE
Accessibility - Heading elements are not in a sequentially-descending order on Syllabus Page

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#221]
**Describe the bug**
Heading elements are not in a sequentially-descending order on the Syllabus page. See [here](https://web.dev/heading-order/?utm_source=lighthouse&utm_medium=devtools)

**Failing Elements**
`<h4 class="panel-title"></h4>`

**To Reproduce**
1. Go to vetswhocode.io
2. Run lighthouse report
3. View accessibility report.

**Expected behavior**
The element should be ordered correctly based on the current markup. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by emma-r-slight** [comment]
I'm happy to work on this issue. :)
